Europa/Österreich/Schweiggers.md

```diff
@@ -0,0 +1,24 @@
+# Schweiggers
+
+Schweiggers liegt im Waldviertel in Niederösterreich. Die Fläche der Marktgemeinde umfasst 58,65 Quadratkilometer. 25,04 Prozent der Fläche sind bewaldet.
+
+Zwischen den Ortsteilen Mödershöf und Schweiggers entspringt die Deutsche Thaya, einer von zwei Quellflüssen der Thaya.
+
+## Geschichte
+
+Der Name Schweiggers leitet sich vom althochdeutschen Vornamen Swidger ab und taucht in den Urkunden erstmals um 1182 als „Swichers“ auf. Der Name veränderte sich im Laufe der Jahrhunderte, bis er 1643 als „Schweiggers“ aufscheint.
+
+1394 erhielt die Gemeinde das Marktrecht und hat dadurch einen weitläufigen Marktplatz im Zentrum des Ortes.
+
+1966 wurde die Marktgemeinde Schweiggers mit den Gemeinden Mannshalm (mit den Katastralgemeinden Mannshalm, Streitbach Schwarzenbach und Meinhartsschlag) und Perndorf (mit den KGs Perndorf, Kleinwolfgers und Unterwindhag) zusammengelegt.
+
+1971 erfolgte die zweite Zusammenlegung mit den Gemeinden Sallingstadt (KG Sallingstadt und Walterschlag) sowie Limbach (KG Limbach und Windhof) und Siebenlinden (KG Siebenlinden, Vierlings, Brunnhöf, Bichlhof, Groß Reichenbach und Reinbolden).
+
+## Sehenswürdigkeiten
+
+* Waldviertler Erdäpfelwelt
+* Pfarrkirche Siebenlinden
+
+## Bild
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/c/c5/Rathaus_in_Schweiggers_2018.jpg" alt="Rathaus in Schweiggers\>
```

Comment:
Hier hat sich leider noch immer ein kleiner Fehler eingeschlichen. (Am Ende des "alt" Parameters fehlt ein Anführungszeichen und es ist noch der Backslash statt des Slash verwendet.) 
Dies führt wieder dazu, dass (zumindest bei mir) die Codezeile angezeigt wird. 
Ich habe das mal testweise geändert. Dann wird zumindest nicht mehr die Codezeile angezeigt. Allerdings wird ein Link erzeugt, der zu einer Seite mit foldendem Fehler führt: 

> Content length exceeded

Offenbar liegt das Bild in einer zu großen Auflösung vor. 
Ich habe daher folgenden Link genutzt: 

> https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Rathaus_in_Schweiggers_2018.jpg/640px-Rathaus_in_Schweiggers_2018.jpg
 (640 x 427 Pixel) 

Daraus ergibt sich folgende Zeile, die zumindest bei mir richtig dargestellt wird: 
`<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Rathaus_in_Schweiggers_2018.jpg/640px-Rathaus_in_Schweiggers_2018.jpg" alt="Rathaus in Schweiggers"/>`

Reply:
Danke für die Gelegenheit den ganzen Prozess zu üben.